examples/SwissPaymentSlipFpdf_Orange_Slip.php

```diff
@@ -23,7 +23,7 @@
 $time_start = microtime(true);
 
 // Make sure the classes get auto-loaded
-require __DIR__.'/../vendor/autoload.php';
+$loader = require __DIR__. '/../vendor/autoload.php';
```

Comment:
The reason for making the change was, that it was actually implemented that way in the Tcpdf code base. 
Removed in 534c28e870a49623be78b9b03004c99ba9cacc87